src/python/hooks/file_name_conventions.py

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env python3
-# cspell:words colorama, LIGHTRED, LIGHTBLACK, LIGHTWHITE, LIGHTGREEN
+# cspell:words LIGHTBLACK
+# cspell:words LIGHTGREEN
+# cspell:words LIGHTRED
+# cspell:words LIGHTWHITE
```

Comment:
I've found that `cspell` is case-insensitive, so I just suggest lowercasing these all